packages/compiler/src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -221,24 +212,15 @@ Set the noise for the animal",
       },
     ],
     "name": "Animal",
-    "properties": Array [],
+    "properties": [],
     "sourceUrl": "https://github.com/palantir/documentalist/blob/develop/packages/compiler/src/__tests__/__fixtures__/classes.ts#L19",
   },
-  "Dog": Object {
-    "accessors": Array [
-      Object {
-        "documentation": Object {
-          "contents": Array [
-            "<p>Get the noise of the animal
-Set the noise for the animal</p>
-",
-          ],
-          "contentsRaw": "Get the noise of the animal
-Set the noise for the animal",
-          "metadata": Object {},
-        },
+  "Dog": {
+    "accessors": [
+      {
+        "documentation": undefined,
```

Comment:
need to look into why `get` accessors are no longer getting documented. this is low-priority, though, since we rarely use these in our frontend code bases.

Reply:
actually, it looks like we still have the get & set documentation, but this redundant compound "documentation" for get + set together has been removed. that's fine ✅